scripts/refresh-assets.ts

```diff
@@ -58,6 +61,29 @@ async function copyIcons() {
 		return { fileName: icon, data: svg.data }
 	})
 
+	// Merge svgs into a single file
+
+	const mergedSvgParts = [
+		mergedIconHeader,
+		'<style>',
+		'svg > [id] { display: none; }',
+		'svg > [id]:target { display: inline; }',
+		'</style>',
+	]
+	for (const { fileName, data } of optimizedSvgs) {
+		if (!data.startsWith(mergedIconHeader) || !data.endsWith(mergedIconFooter)) {
+			nicelog('SVG does not match expected format for merging:', fileName)
+			process.exit(1)
+		}
+
+		const id = fileName.replace('.svg', '')
+		const svgContent = data.slice(mergedIconHeader.length, -mergedIconFooter.length)
+		mergedSvgParts.push(`<g id="${id}">${svgContent}</g>`)
+	}
+	mergedSvgParts.push(mergedIconFooter)
+	const mergedSvg = optimize(mergedSvgParts.join('\n')).data
```

Comment:
this is prbly fine but just checking - `optimize` is a bit of a black box to me - now that the assets are merged, it doesn't over-eagerly optimize/weirdly combine two assets that are perhaps side-by-side?